src/chrome/content/rules/Hupu.xml

```diff
@@ -0,0 +1,58 @@
+<!--
+    Nonfunctional (403 Forbidden):
+        - photo.hupu.com
+        - run.hupu.com
+        - tennis.hupu.com
+        - www-backend.hupu.com
+        - zb.hupu.com
```

Comment:
Above subdomains aren't 403 (anymore), and you can add www-backend.hupu.com as a target.